### PR TITLE
Introduce NoopLR

### PR DESCRIPTION
**Pull request by cbalioglu**
A nit PR that introduces a no-op learning rate scheduler.

### CODE REVIEW

File: src/fairseq2/optim/lr_scheduler.py [hunk 1]
```diff
@@ -41,6 +41,18 @@ def _compute_lrs(self) -> List[float]:
         """Compute the learning rate of each parameter group."""
 
 
+@final
+class NoopLR(AbstractLRScheduler):
```
[the remark this answers is not in the document]
Comment (cbalioglu) [reply src/fairseq2/optim/lr_scheduler.py:45]: You can mimic the same behavior with `ConstantLR(optimizer, factor=1.0)`, but it feels a bit off.
